### PR TITLE
Two term approx and conservativeness tests. 

### PR DESCRIPTION
**Pull request by 1ozturkbe**
Lots of features are completely untested. I will keep tabs on what needs

### CODE REVIEW

**Comment by bqpd** [comment]
We could look into adding coverage to these unit tests.

**Comment by 1ozturkbe** [comment]
That would be lovely. How does one do that?

**Comment by bqpd** [comment]
ugh after trying for an hour to get it working it looks like it needs the tests to be set up as a package (as in gpkit), not from the TESTS file

**Comment by 1ozturkbe** [comment]
Is that as simple as importing the right packages? Because if so, it seems like low hanging fruit.

**Comment by bqpd** [comment]
ha, I wish! but no. simplest way to do it would be to rewrite each file in TESTS as a test method that explicitly imports it.

**Comment by 1ozturkbe** [comment]
I guess I'll check out gpkit and try to replicate it.

File: robust/equivalent_posynomials.py [hunk 1]
```diff
@@ -83,13 +85,22 @@ def __init__(self, p, m, simple_model, dependent_uncertainties):
         monomials = p.chop()
         for i, mon_list in enumerate(uncertain_vars_exps_mons):
             if len(mon_list) > 1 and uncertain_vars_exps[i]:
+                # Really clunky way of creating monomials from exponents...
+                # Should streamline/speed up in the future.
                 new_no_data_posynomial = 0
+                hmap = NomialMap({HashVector(uncertain_vars_exps[i]): 1.0})
+                unitarr = [k.units**v for k, v in uncertain_vars_exps[i].items() if k.units is not None]
```
Comment (bqpd) [inline robust/equivalent_posynomials.py]: could just be [k.units**v for k, v in uncertain_vars_exps[i].items() if k.units]

Comment (1ozturkbe) [reply robust/equivalent_posynomials.py]: haha thanks Ned, will do!

**Pull request merged**